providers/edge3/src/airflow/providers/edge3/models/edge_worker.py

```diff
@@ -27,13 +27,18 @@
 
 from airflow.exceptions import AirflowException
 from airflow.models.base import Base
+from airflow.providers.edge3.version_compat import AIRFLOW_V_3_1_PLUS
 from airflow.stats import Stats
-from airflow.utils import timezone
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.providers_configuration_loader import providers_configuration_loaded
 from airflow.utils.session import NEW_SESSION, provide_session
 from airflow.utils.sqlalchemy import UtcDateTime
 
+if AIRFLOW_V_3_1_PLUS:
+    from airflow.sdk import timezone
+else:
+    from airflow.utils import timezone  # type: ignore[attr-defined,no-redef]
```

Comment:
```suggestion
try:
    from airflow.sdk import timezone
except ImportError:
    from airflow.utils import timezone  # type: ignore[attr-defined,no-redef]
```

Better not to use version checks, especially as this an import from TaskSDK, which is working torwards independence from the Airflow version.

Reply:
Fixed.